openfasoc/MLoptimization/sample_spec.yaml

```diff
@@ -0,0 +1,100 @@
+{'gain_min': [2486222284.8408804,
+ 2595551369.5626364,
+ 2309495771.9680934,
+ 1145774876.7498627,
+ 2079516450.1469443,
+ 2391451213.479562,
+ 2676893053.18551,
+ 2675006145.1962533,
+ 2015194984.3624444,
+ 2278788513.705864,
+ 2205636091.187187,
+ 1506492988.8309956,
+ 1112077088.809704,
+ 2707199150.1559224,
+ 2903609807.9496045,
+ 2062437443.5261226,
+ 1403056317.139003,
+ 2835219051.771353,
+ 2925021136.524023,
+ 1886275826.821474,
+ 2157488694.831938,
+ 2641455527.072985,
+ 2402196770.75093,
+ 1199232764.3421414,
+ 2976991031.6658754,
+ 1646354535.6840703,
+ 2704204845.6734324,
+ 2716618295.460021,
+ 2413570061.996591,
+ 2121921633.2690156,
+ 2303933548.53714,
+ 2028867678.9123795,
+ 2719297024.9283357,
+ 2173990079.053155,
+ 1445459741.8651175,
+ 2472330865.96015,
+ 1001989471.0531384,
+ 2759910539.938794,
+ 1536809631.1164005,
+ 1135029016.3512793,
+ 1588011527.4173698,
+ 2049869876.6713438,
+ 2427789734.566804,
+ 1047544300.167047,
+ 1914004908.9098907,
+ 1393488128.8226607,
+ 2559241796.7458553,
+ 1565752673.493743,
+ 1295775028.0897965,
+ 1918300980.2757072],
+ 'FOM': [500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0,
+ 500000000000.0]}
```

Comment:
I think it's a better idea to provide the script that generates this yaml file instead of uploading raw data like this?
`gain_min` is from one of the earlier runs, I think it's deprecated and need to be updated.
`FOM` is also very old, need to be updated.